test/REQUIRE

```diff
@@ -1 +1,2 @@
-FactCheck
\ No newline at end of file
+FactCheck
+DataStructures

```

Comment:
This isn't needed here. If it's in the main `REQUIRE` file it doesn't need to be here.